assets/tests/consolidated_1_1_tests.yaml

```diff
@@ -2318,7 +2514,27 @@ tests:
               relation: viewer
             expectation:
               - document:public
-
+        listUsersAssertions:
+          - request:
+              filters:
+                - user
+              object: document:public
+              relation: viewer
+            expectation:
+              - user:*
```

Comment:
Consider adding a tuple `document:public#viewer@user:jon` and also asserting you get `[user:*, user:jon]` here. This gives us confidence we follow both paths and yield both the wildcard and the explicit subject.